src/_lib/modules/charts/axis/x-axis.directive.ts

```diff
@@ -4,6 +4,7 @@ import { AxisBase } from './axis-base';
 import { ScaleTime, scaleTime } from 'd3-scale';
 import { ChartComponent } from '../chart.component';
 import { ChartUtils } from "../chart.utils";
+import {select, Selection} from 'd3-selection';
```

Comment:
Sorry...spacing before and after brackets

Reply:
we need a common lint file :(

Reply:
Yes, we really do